### DISCUSSION TITLE
ElasticSearch JSON query to Pydantic Schema for FastAPI

### DISCUSSION THREAD

**Discussion by chaman9** [fastapi/fastapi#3731] [Questions]
How can we convert ElasticSearch JSON query to pydantic Schema model for FastAPI. 

I am using FastAPI in the backend and have different filters on frontend for which I used elasticsearch (database) query (Json format), now I need to write the pydantic schema to display it to the swagger UI?

Query:

{
"query": {
    "bool": {
      "must": [  
          {
          "multi_match": {
            "query": "JOHN AND ANDY",
            "fields": [FIRST_NAME]
          }
        },
        {
          "multi_match": {
            "query": "SAM",
            "fields": [LAST_NAME]
          }
        },
        {
          "range": {
            "height": {
              "gte": 0,
              "lte": 100
            }
          }
        },
        {
          "range": {
            "length1": {
              "gte": 0,
              "lte": 100
            }
          }
          },
          
          {
          "range": {
            "length2": {
              "gte": 0,
              "lte": 100
            }
          }
        },
        {
          "range": {
            "length3": {
              "gte": 0,
              "lte": 100
            }
          }
        },
        {
          "range": {
            "length4": {
              "gte": 0,
              "lte": 100
            }
          }
        }
      ]
    }
  }
}

Can you please provide me more insights into it? Appreciate your help

**Comment by ghandic** [comment]
Thank me later - https://jsontopydantic.com
